docs/functions/util.rst

```diff
@@ -72,12 +72,12 @@ These functions are designed to be used by themes.
    some spaces will be inserted after the last occurrence of *fillstring*,
    so as to match the exact width of the terminal.
 
-   *fillstring* may contains escaped sequences (such as colors).
+   *fillstring* may NOT contains escaped sequences (such as colors).
 
-   .. note:: If *fillstring* have multiple characters and *splitends* is 1,
-             then **the last occurrence** of *fillstring*
-             is stripped of any escaped sequences
-             and only its printable characters are inserted.
+  .. note:: Any escaped sequence in *fillstring* will be removed automatically.
+
+   *fillprefix* and *fillsuffix* may be used to add escaped sequences at the
+   beginning and, respectively, the end of the filling sequence.
```

Comment:
This is confusing. I can't put escape sequences in `fillstring`, but if I do, they will be removed?

I think this would make it more clear:
```rst
   Any escaped sequence in *fillstring* will be removed automatically.
   *fillprefix* and *fillsuffix* may be used to add escaped sequences at the
   beginning and end, respectively, of the filling sequence.
```